src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -37,8 +37,9 @@ <h5><span jhiTranslate="artemisApp.studentExamDetail.student">Student</span></h5
         <h5><span jhiTranslate="artemisApp.studentExams.workingTime" [ngbTooltip]="getWorkingTimeToolTip()">Working time</span></h5>
         <form class="d-flex" [formGroup]="workingTimeForm" (ngSubmit)="saveWorkingTime()">
             <div class="me-2" [ngbTooltip]="getWorkingTimeToolTip()">
-                <input formControlName="minutes" type="number" style="width: 60px; height: 35px" class="text-center" min="0" step="1" required /> h
-                <input formControlName="seconds" type="number" style="width: 60px; height: 35px" class="text-center" min="0" max="59" step="1" required /> min
+                <input formControlName="hours" type="number" style="width: 60px; height: 35px" class="text-center" min="0" step="1" required /> h
+                <input formControlName="minutes" type="number" style="width: 60px; height: 35px" class="text-center" min="0" max="59" step="1" required /> min
+                <input formControlName="seconds" type="number" style="width: 60px; height: 35px" class="text-center" min="0" max="59" step="1" required /> s
```

Comment:
It might make sense to introduce a `css` class for the style? 
e.g. 
```
.durationInput {
    width: 60px;
    height: 35px;
}
```

```suggestion
                <input formControlName="hours" type="number" class="text-center durationInput " min="0" step="1" required /> h
                <input formControlName="minutes" type="number" class="text-center durationInput " min="0" max="59" step="1" required /> min
                <input formControlName="seconds" type="number" class="text-center durationInput " min="0" max="59" step="1" required /> s
```

I think that it would be better to use bootstrap input fields instead of styling raw input fields, but from a short look we are using raw input fields within the application so far - so if we would want to change this it would be another PR.

Reply:
I think I’m going to change this form completely anyway in the implementation for #4581 to be able to make the value display easily dynamic when changing either the percent or the duration values. So I’m not going to change it here, but I’ll keep it in mind for the next PR.

At the moment I think that something like in the [Bootstrap documenation for input groups](https://getbootstrap.com/docs/5.1/forms/input-group/#basic-example) for the ‘Username @ Server’ would look nice to group the Xh Ymin Zsec inputs.